backends/platform/atari/readme.txt

```diff
@@ -0,0 +1,422 @@
+ScummVM
+=======
+
+This is a new port of ScummVM (https://www.scummvm.org), a program which allows
+you to run certain classic graphical adventure and role-playing games, provided
+you already have their data files.
+
+You can find a full list with details on which games are supported and how well
+on the compatibility page: https://www.scummvm.org/compatibility.
+
+
+Yet another port?
+-----------------
+
+Yes, I am aware of the official Atari/FreeMiNT port done by KeithS over the
+years (https://docs.scummvm.org/en/v2.6.1/other_platforms/atari.html). It is
+even updated every release and put on the official ScummVM website. That port
+is basically just a recompiled SDL backend for our platform - that has some
+advantages (works in GEM, can be easily compiled for the FireBee etc) but also
+a few disadvantages, like:
+
+- It's huge, massively huge, about 51 MiB. It contains about every engine,
+  every library which is available to link on our platform. Most of that stuff
+  is not needed, see my next point.
+
+- A good half of the features are basically useless on our platform (like
+  hi-res 16bpp graphics, software synthesizers, scalers, real-time software
+  MP3/OGG/FLAC playback etc) ... there's basically no horsepower left for such
+  perks!
+
+- It's SDL based. That is good for the advantages listed above but it has its
+  prize: for one, it makes the whole thing even slower because SDL is yet
+  another layer of abstraction. And what is worse, the way the SDL backend is
+  used on higher-end platforms doesn't have to be exactly optimal on ours
+  (where every additional copying between buffers, every additional redraw,
+  hurts). Also, sometimes our SDL port tries to emulate certain features (i.e.
+  audio threads) in a way which can easily crash (e.g. putting them in a timer
+  callback which has only limited time for running).
+
+- It's sparsely tested. Due to the 'features' listed above, not many people use
```

Comment:
Because we limit scope only to 8bpp games, it opens a door to more thorough testing and there is a certain interest in this in the community.